modules/qna/src/backend/index.ts

```diff
@@ -54,13 +54,33 @@ const onFlowChanged = async (bp: typeof sdk, botId: string, newFlow: sdk.Flow) =
   }
 }
 
+const onElementChanged = async (bp, botId, action, element) => {
+  const qnaStorage = await botScopedStorage.get(botId)
+  const questions = await qnaStorage.getQuestions({ question: '', categories: [] }, { limit: 0, offset: 0 })
+  // Update all questions that have this contentElement as answer
+  if (action === 'delete') {
+    const affectedItems = questions.items.filter(item =>
```

Comment:
the first filter does nothing right? it will always return true since an empty array is thruthy

Reply:
Yes, we should filter only answers that have the contentElement as answer, i will take a deeper look at that